docs/api/desktop-capturer.md

```diff
@@ -100,8 +100,23 @@ Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`Desktop
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
-## Caveats
+## System Audio Capture (macOS 13+)
 
-`navigator.mediaDevices.getUserMedia` does not work on macOS for audio capture due to a fundamental limitation whereby apps that want to access the system's audio require a [signed kernel extension](https://developer.apple.com/library/archive/documentation/Security/Conceptual/System_Integrity_Protection_Guide/KernelExtensions/KernelExtensions.html). Chromium, and by extension Electron, does not provide this.
+ On macOS 13+, `navigator.mediaDevices.getDisplayMedia` can be used to capture system audio
+ if the [`setDisplayMediaRequestHandler`](./session.md#sessetdisplaymediarequesthandlerhandler-opts) `callback` 
+ is passed a stream that has `audio: 'loopback'`. See the [example](./desktop-capturer.md) above.
+
+> [!NOTE]
+> Due to ongoing changes in Chromium, capturing system audio is experimental on macOS 15+ and requires the 
+> following command line switch to be appended:
+>
+> ```js
+> // main.js
+> app.commandLine.appendSwitch('enable-features', 'MacSckSystemAudioLoopbackOverride');
+> ```
+
+## System Audio Capture (macOS 12.3 or lower)
```

Comment:
What about macOS >12.3 <=13? Should we just say "macOS 12 or lower"?

Same in the line below.